### PR TITLE
Rebuild MultiBrand theme on top of master branch

### PR DESCRIPTION
**Pull request by ellimist**
Rebuild the multibrand-branch on top of master.

### CODE REVIEW

File: templates/search_results.hbs [hunk 1]
```diff
@@ -5,103 +5,102 @@
...
+                {{#if comment_count}}
+                  <span class="search-results-meta-count">
+                    {{comment_count}}
+                  </span>
+                {{/if}}
+              </div>
               <ul class="meta-group">
                 <li>
-                  <ol class="breadcrumbs search-result-breadcrumbs">
+                  <ol class="breadcrumbs search-results-breadcrumbs">
```
Comment (VilhelmNielsen) [inline templates/search_results.hbs]: These class renames break the styling here https://github.com/zendesk/copenhagen_theme/blob/bc646972db322ba09bd877602ef36465d51eaed1/styles/_search_results.scss#L100

File: templates/search_results.hbs [hunk 2]
```diff
@@ -5,103 +5,102 @@
...
+        </section>
+      {{/if}}
+    </section>
+
     <section class="search-results-column">
-      <h2 class="search-results-subheading">
-        {{t 'knowledge_base'}}
-      </h2>
-      {{#if article_results}}
+      <h3 class="results-subheading">
```
Comment (VilhelmNielsen) [inline templates/search_results.hbs]: This should be an `h1`

File: templates/search_results.hbs [hunk 3]
```diff
@@ -5,103 +5,102 @@
...
         <ul class="search-results-list">
-          {{#each article_results}}
-            <li class="search-result">
-              <a href="{{url}}" class="search-result-link">{{title}}</a>
-              {{#if vote_sum}}
-                <span class="search-result-votes meta-count">{{vote_sum}}</span>
-              {{/if}}
+          {{#each results}}
+            <li class="search-results-list-item result-{{type}}">
+              <a href="{{url}}" class="results-list-item-link">{{title}}</a>
```
Comment (VilhelmNielsen) [inline templates/search_results.hbs]: Nit: For a11y, the titles should be encapsulated in a heading, e.g. `h2` to establish hierarchy. And each result could be wrapped in a `article` tag.

**Comment by ellimist** [comment]
Thanks for the comments @VilhelmNielsen 🙏 I addressed them.

**Comment by ellimist** [comment]
Preview:

### Desktop:

image

### Mobile

Closed filters:
image

Open Filters:
image

**Pull request closed without merging**